changelog.txt

```diff
@@ -20,6 +20,7 @@ V.15.0.0
 - [MINOR] Add checkMode method to msaltestapp infra (#2141)
 - [MINOR] Update TokenRequest.java with NAA params (#2143)
 - [MINOR] Add new apk name to BrokerHost infra (#2152)
+- [MINOR] Add two extra steps for handleFirstRun method in OneAuthTestApp infra (#2161)
```

Comment:
I don't think a changelog entry is needed for UI Automation testing changes

Reply:
I've removed the changelog change and added no-changelog label